devops/llvm-anybuild.yml

```diff
@@ -10,19 +10,19 @@ parameters:
   # extra command-line args to pass to AnyBuild
   - name: ExtraAnyBuildArgs
     type: string
-    default: '--Epoch 1'
+    default: '--Epoch 1 --WhyCacheMiss --WhyCacheMissOptions CacheDataStoreKey=LLVMVerona'
   # LLVM commit hash to build
   - name: GOOD_HASH
     type: string
-    default: '73811d3'
+    default: 9f33943ee01
```

Comment:
On our side, we don't have a default value on the yaml file, and leave it as a variable in the build. This is meant to be a conscious change and error out if the value is not set, rather than build an older version by mistake.

Recurrent builds can still use the variable value, if driven by DevOps, so it shouldn't change stress testing either.

Reply:
changed to a variable and added a check that it is set in DevOps